agent/config/config_unix.go

```diff
@@ -28,6 +28,7 @@ func DefaultConfig() Config {
 		ReservedPorts:               []uint16{SSHPort, DockerReservedPort, DockerReservedSSLPort, AgentIntrospectionPort, AgentCredentialsPort},
 		ReservedPortsUDP:            []uint16{},
 		DataDir:                     "/data/",
+		InstanceDataDir:             "/var/lib/ecs",
```

Comment:
Currently in my implementation, many of the paths are constructed from the agent-container file path system (For UNIX). All these paths are formed by DataDir + "/metadata/" + everything else. This includes the function we use to get the metadata file's intended file path. InstanceDataDir is the directory on the host instance that is mapped to /data/ in the container and we use this only as a prefix for building the mount path to the metadata.